### PR TITLE
Update view.html

### PR DESCRIPTION
**Pull request by jesseke55**
Fixed a couple typo`s

### CODE REVIEW

File: app/views/admin/server/view.html [hunk 1]
```diff
@@ -263,21 +263,21 @@ <h3 class="nopad">Connection Information</h3><hr />
 		<div class="tab-pane" id="installer">
 			<h3>Install Progress</h3><hr />
 				<div class="alert alert-danger" id="socket_error" style="display:none;">
-					We were unable to connect to the main server, there may be network issues currently. This message will disappear when we are able to connect, you do not need to refresh.
+					We were are to connect to the main server, there may be network issues currently. This message will disappear when we are able to connect, you do not need to refresh.
```
Comment (Omeryl) [inline app/views/admin/server/view.html]: You made a typo instead of fixing it..

Comment (jesseke55) [inline app/views/admin/server/view.html]: I`m gonna fix that, sorry about that.

**Comment by Omeryl** [comment]
Furthermore, I don't believe they're making any changes to this branch any more, in favor of their new version they're working on.

**Comment by jesseke55** [comment]
We will see it then.

**Comment by akiva3577** [comment]
If I'm not mistaken they first want to fix the big bugs before updating to v1

**Comment by puffrfish** [comment]
We do have certain guidelines for what we are accepting pull requests for right now which should have been shown to you with the PR template.

I do appreciate the effort, but some of the changes you are making are not actually fixing anything. "We were unable to connect to the main server" is perfectly fine.

The other two changes I will accept, but I don't think it will be helpful to see more of these. I will run through the panel and check for misspellings and grammatical errors before the next release.

**Comment by jesseke55** [comment]
Oh ok

**Comment by puffrfish** [comment]
What I was saying was that if you take out the change to "We were unable to connect to the main server" I will accept the PR. Unless you'd rather not do that.

**Comment by jesseke55** [comment]
Updated.

**Comment by puffrfish** [comment]
Thanks!

**Pull request merged**